submissions/chabuChabu/javaScript/signup.js

```diff
@@ -0,0 +1,121 @@
+let submitForm = document.getElementById('validateSignUp'),
+    firstName = document.getElementById('firstName'),
+    lastName = document.getElementById('lastName'),
+    phoneNumber = document.getElementById('phoneNumber'),
+    emailAddress = document.getElementById('emailAddress'),
+    password = document.getElementById('password'),
+    confirmPassword = document.getElementById('confirmPassword')
+;
+
+function required(field, event) {
+    if(field.value === '') {
+        field.nextElementSibling.innerHTML = "This is a required field";
+        event.preventDefault();
+        return false;
+    } else {
+        field.nextElementSibling.innerHTML = "";
+        return true;
+    }
+}
+
+function fNameValidation(event){
+    let fNameRegex = /^[A-Za-z]+$/;
+    if(!firstName.value.match(fNameRegex)){
+        firstName.nextElementSibling.innerHTML = "Please enter your first name";
+        event.preventDefault();
+        return false;
+    }else{
+        firstName.nextElementSibling.innerHTML = "";
+        return true;
+    }
+}
+
+function lNameValidation (event){
```

Comment:
`fNameValidation` and `lNameValidation` do the same thing. the only difference is the `regex`, `field to be validated` and `error message display`. Kindly reuse the same function and passed the differences that they have as a parameter